src/get-in-touch.njk

```diff
@@ -0,0 +1,16 @@
+<div class="govuk-o-width-container app-c-site-width-container">
+    <div class="govuk-o-grid govuk-o-main-wrapper">
+      <div class="govuk-o-grid__item govuk-o-grid__item--two-thirds">
+         <h1 class="govuk-heading-xl">Get in touch</h1>
+
+         <p class="govuk-body-l">If you’ve got a question, idea or suggestion get in touch with the Design System team.</p>
+
+         <h2 class="govuk-heading-l">Private beta Slack channel</h2>
+
+         <p class="govuk-body">Use <a href="https://ukgovernmentdigital.slack.com/messages/govuk-design-system">#govuk-design-system</a>, on cross-government Slack (<a href="slack://channel?team=T04V6EBTR&id=C6DMEH5R6">open in app</a>)</a></p>
```

Comment:
I think the comma here is a bit odd:

Use <a href="#">#govuk-design-system</a>, on cross-government Slack (<a href="#">open in app</a>)

@amyhupe shall we take it out?